tests/Functional/BreadcrumbRenderTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+/**
+ * Tests that breadcrumbs are rendered correctly.
+ *
+ * @group batch1
+ */
+class BreadcrumbRenderTest extends ContentRenderTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'oe_theme_helper',
+    'path',
+    'oe_theme_content_news',
+    'block',
+  ];
+
+  /**
+   * Tests that the breadcrumbs are rendered correctly for news content type.
+   */
+  public function testNewsBreadcrumbRendering(): void {
```

Comment:
The scope of this test is to test the caching, not the rendering, and caching in general, not of the news content type. If I see this method, I will never imagine that we are trying to cover the caching of the last breadcrumb item.
Let's name it `testBreadcrumbCaching`.